SimCalorimetry/HcalTrigPrimAlgos/interface/HcalTriggerPrimitiveAlgo.h

```diff
@@ -124,7 +124,7 @@ class HcalTriggerPrimitiveAlgo {
   double theThreshold;
   bool peakfind_;
   std::vector<double> weights_;
-  std::map<int, std::vector<double>> weightsQIE11_;
+  std::map<unsigned int, std::vector<double>> weightsQIE11_;
```

Comment:
why a map?
would just std::array<std::array<double,2>,28> not do it?

Reply:
No profound reason to use a map, just out comfort. I have adopted the array of arrays approach.